### PR TITLE
Sei Github Documentation

### PR DESCRIPTION
**Pull request by Kbhat1**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+
+Sei Network is the first orderbook-specific L1 blockchain. The chain emphasizes reliability, security and high throughput above all else, enabling an entirely new echelon of ultra-high performance DeFi products built on top. Sei's on-chain CLOB and matching engine provides deep liquidity and price-time-priority matching for traders and apps. Apps built on Sei benefit from built-in orderbook infrastructure, deep liquidity, and a fully decentralized matching service. Users benefit from this exchange model with the ability to select price, size, and direction of their trades coupled with MEV protection.
+
 # seichain
-**seichain** is a blockchain built using Cosmos SDK and Tendermint and created with [Starport](https://starport.com).
+**seichain** is a blockchain built using Cosmos SDK and Tendermint and created with [Starport](https://starport.com). It is built using the Cosmos SDK and Tendermint core, and features a built-in central limit orderbook (CLOB) module. Decentralized applications building on Sei can build on top of the CLOB, and other Cosmos-based blockchains can leverage Sei's CLOB as a shared liquidity hub and create markets for any asset. Sei Shared Liquidity Model
```
Comment (LCyson) [inline readme.md]: Can we remove the created with starport part here

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+Designed with developers and users in mind, Sei serves as the infrastructure and shared liquidity hub for the next generation of DeFi. Apps can easily plug-and-play to trade on Sei orderbook infrastructure and access pooled liquidity from other apps. To prioritize developer experience, Sei Network has integrated the wasmd module to support CosmWasm smart contracts.
 
 ## Get started
 You may use starport to run the chain, but typically we have our own customizations that require using an internal tool (seid). Both methods are shown below.
```
Comment (LCyson) [inline readme.md]: can we remove the technical instruction here, we can put a gitbook link here instead

**Comment by LCyson** [comment]
also can we add skeletons (specs) to each of the module? we can fill them out later 
https://github.com/osmosis-labs/osmosis/tree/main/x/mint/spec

**Review by LCyson — APPROVED** [review]

**Comment by Kbhat1** [comment]
@LCyson as mentioned in our chat, going to first merge this base skeleton high level documentation, have Chalabi merge his testnet readme documentation and then add more documentation for the module level

**Pull request merged**